flow/instrumentation.h

```diff
@@ -16,14 +16,35 @@ namespace flutter {
 
 class Stopwatch {
  public:
-  explicit Stopwatch(fml::Milliseconds frame_budget = fml::kDefaultFrameBudget);
+  /// The update type that is passed to the Stopwatch's constructor.
+  enum FrameBudgetUpdateType {
+    /// Update the latest frame budget value from refresh rate everytime.
+    /// See: `DisplayManager::GetMainDisplayRefreshRate`
+    kUpdateEverytime,
+    /// Using the default value `fml::kDefaultFrameBudget` (60 fps)
+    kOnShotValue,
+  };
```

Comment:
Do we really need this? Why not just _always_ pass in the rasterizer as a delegate and then call `GetFrameBudget` on it every time?

Reply:
> Do we really need this? Why not just _always_ pass in the rasterizer as a delegate and then call `GetFrameBudget` on it every time?

@dnfield   I tried to make only a delegate in the `GetFrameBudget()` logic, and I need to find a place and memory to store the `FixedRefreshRateDelegate` instance. A "0 - params" constructor of `CompositorContext` and some unit tests need the fixed version like `FixedRefreshRateStopWatch`.  Do you have any advice to make the code better and more pretty?